### ISSUE TITLE
Crop grid partly covered by transparent navigation bar

### ISSUE THREAD

**Issue by koloritcm** [kishikawakatsumi/PEPhotoCropEditor#9]
I push the the crop controller onto the UIImagePickerController (that is a UINavigationController). The existing navigation bar is transparent, and therefor I guess it thinks that it should use the area under the navigation bar. So the result is that the crop view is partly covered by the navigation bar.

**Comment by brockboland** [comment]
I'm seeing this bug too, particularly on tall, narrow images:

![ios simulator-1](https://f.cloud.github.com/assets/163838/1829946/30c10c2e-7312-11e3-8722-b94459417f61.png)

Based on an [answer on StackOverflow](http://stackoverflow.com/a/18953585/2185), I found that turning off the translucence of the navigation bar (at the top) and the toolbar (at the bottom) in `viewDidLoad:` fixes the layout problem:

``` objective-c
self.navigationController.navigationBar.translucent = NO;
self.navigationController.toolbar.translucent = NO;
```

If you only set `self.navigationController.navigationBar.translucent` (as mentioned in the SO answer), the view still extends to the bottom further, which makes the whole thing look lopsided:

![ios simulator-2](https://f.cloud.github.com/assets/163838/1830194/bceeed3e-732a-11e3-8d24-6170c44aa560.png)

**Comment by brockboland** [comment]
Opened pull request #25 with a fix.

**Comment by brockboland** [comment]
This was fixed in d88714f17fbac38db39d8dac723bd9f2698efc75, and the fix is included in v1.2.3. Please close this issue.
